src/EnergyPlus/FluidProperties.cc

```diff
@@ -243,181 +218,397 @@ namespace FluidProperties {
-             1.0016e-3, 0.8901e-3, 0.7974e-3, 0.7193e-3, 0.653e-3,  0.5961e-3, 0.5468e-3, 0.504e-3,  0.4664e-3, 0.4332e-3, 0.4039e-3,
-             0.3777e-3, 0.3543e-3, 0.3333e-3, 0.3144e-3, 0.2973e-3, 0.2817e-3, 0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3}); // in Pa-s
+        constexpr std::array<Real64, DefaultNumGlyTemps> DefaultWaterViscData = {
+            0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    1.7912e-3, 1.5183e-3, 1.306e-3,  1.1376e-3,
+            1.0016e-3, 0.8901e-3, 0.7974e-3, 0.7193e-3, 0.653e-3,  0.5961e-3, 0.5468e-3, 0.504e-3,  0.4664e-3, 0.4332e-3, 0.4039e-3,
+            0.3777e-3, 0.3543e-3, 0.3333e-3, 0.3144e-3, 0.2973e-3, 0.2817e-3, 0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3,    0.0e-3}; // in Pa-s
 
-        static Array1D<Real64> const DefaultWaterRhoData(DefaultNumGlyTemps,
-                                                         {0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   999.8, 999.9, 999.7, 999.1,
-                                                          998.2, 997.0, 995.6, 994.0, 992.2, 990.2, 988.0, 985.7, 983.2, 980.5, 977.7,
-                                                          974.8, 971.8, 968.6, 965.3, 961.9, 958.3, 0.0,   0.0,   0.0,   0.0,   0.0}); // in kg/m3
+        constexpr std::array<Real64, DefaultNumGlyTemps> DefaultWaterRhoData = {
+            0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   999.8, 999.9, 999.7, 999.1, 998.2, 997.0, 995.6, 994.0, 992.2, 990.2,
+            988.0, 985.7, 983.2, 980.5, 977.7, 974.8, 971.8, 968.6, 965.3, 961.9, 958.3, 0.0,   0.0,   0.0,   0.0,   0.0}; // in kg/m3
 
-        static Array1D<Real64> const DefaultWaterCondData(DefaultNumGlyTemps, {0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.0,    0.561,  0.5705,
-                                                                               0.58,   0.5893, 0.5984, 0.6072, 0.6155, 0.6233, 0.6306, 0.6373, 0.6436,
-                                                                               0.6492, 0.6543, 0.659,  0.6631, 0.6668, 0.67,   0.6728, 0.6753, 0.6773,
-                                                                               0.6791, 0.0,    0.0,    0.0,    0.0,    0.0}); // in W/mK
+        constexpr std::array<Real64, DefaultNumGlyTemps> DefaultWaterCondData = {
+            0.0,    0.0,    0.0,    0.0,   0.0,    0.0,    0.0,  0.561,  0.5705, 0.58,   0.5893, 0.5984, 0.6072, 0.6155, 0.6233, 0.6306, 0.6373,
+            0.6436, 0.6492, 0.6543, 0.659, 0.6631, 0.6668, 0.67, 0.6728, 0.6753, 0.6773, 0.6791, 0.0,    0.0,    0.0,    0.0,    0.0}; // in W/mK
 
         // Ethylene Glycol Data: Specific Heat in J/(kg-k)
-        static Array2D<Real64> const DefaultEthGlyCpData(
-            DefaultNumGlyTemps, DefaultNumGlyConcs, std::bind(DefaultEthGlyCpData_initializer, _1, DefaultWaterCpData));
+
+        constexpr std::array<std::array<Real64, DefaultNumGlyTemps>, DefaultNumGlyConcs> DefaultEthGlyCpData = {
```

Comment:
`static Array2D<Real64> const` to `std::array<std::array<Real64, 2D array number of rows>, 2D array number of columns>`. Main point: had to specify number of rows and columns when doing this conversion.